### DISCUSSION TITLE
Possible python script return object?

### DISCUSSION THREAD

**Discussion by royalmaster960** [windmill-labs/windmill#1728] [Questions] [answered]
i am using windmill flow and my python script return object which class init. and i want to reuse object class at other flow. 
![image](https://github.com/windmill-labs/windmill/assets/110345381/f625ab29-979f-4053-a336-c5221f566a73)

I am checking result of step 1. The output return string object instead of metabase_api object 
![image](https://github.com/windmill-labs/windmill/assets/110345381/a8e1699d-e122-4a44-a37f-153a77382907)

**Comment by rubenfiszel** [comment, marked as answer]
You cannot just return an object since every output will be serialized to json, however you can likely pickle it. See: https://jsonpickle.github.io/

**Reply by royalmaster960** [reply]
Thanks bro. The best solution now.
